### PR TITLE
J3RN's solutions

### PR DESCRIPTION
**Pull request by J3RN**
I'd love feedback! Nesting `if`s always makes me uncomfortable.

### CODE REVIEW

**Comment by degoeym** [comment]
Remember how I asked you about explicit returns in your grade method in the other challenge? Well, look into utilizing explicit returns in Ruby on a contextual basis for this challenge to not have so many nested If statements. :stuck_out_tongue: 

Overall it works, which is good for the purposes of completing the challenge, but it can definitely be a lot cleaner/more "Ruby".

**Comment by J3RN** [comment]
My `return` statements are probably a result of writing too much C. In any event, there's only one `if` statement now!

**Comment by nicolasmccurdy** [comment]
Nice, `deaf_grandma.rb` looks cleaner now.

File: lib/deaf_grandma.rb [hunk 1]
```diff
@@ -19,9 +19,13 @@ def run!
 
 
   def speak(input)
-
-    #Implement your code here <<<<<<<<<
-
+    if input == input.upcase
+      @bye_counter += 1 if input == "BYE"
+      return "SEE YOU LATER SONNY!" if @bye_counter == 3
+      "NOT SINCE 1964"
```
Comment (nicolasmccurdy) [inline lib/deaf_grandma.rb]: You can simplify the top two lines with the ternary operator.

**Comment by J3RN** [comment]
Ternary solution now in place!

**Comment by nicolasmccurdy** [comment]
:+1:

**Pull request closed without merging**